operator/v1/0000_70_dns-operator_00.crd.yaml

```diff
@@ -42,6 +42,39 @@ spec:
             description: spec is the specification of the desired behavior of the
               DNS.
             properties:
+              cache:
+                description: 'cache describes the caching configuration that applies
+                  to all server blocks listed in the Corefile. This field allows a
+                  cluster admin to optionally configure: * positiveTTL which is a
+                  duration for which positive responses should be cached. * negativeTTL
+                  which is a duration for which negative responses should be cached.
+                  If this is not configured, OpenShift will configure positive and
+                  negative caching with a default value that is subject to change.
+                  At the time of writing, the default positiveTTL is 900 seconds and
+                  the default negativeTTL is 30 seconds or as noted in the respective
+                  Corefile for your version of OpenShift.'
+                properties:
+                  denialTTL:
```

Comment:
Looks like this needs to be regenerated.

Reply:
Fixed